pages/about/code-of-conduct.vue

```diff
@@ -272,6 +278,8 @@ export default {
     }
     .sub2_img {
         @apply flex;
+        margin-left: -1.5rem;
+        margin-right: -1.5rem;
```

Comment:
Maybe we can write into `@apply` like this:
```
@apply my-[-1.5rem]
```

Reply:
OK